Life and death update

Yzzuwyzzuf opened the pull request:
## About The Pull Request

adds new thoughts for categories like leaders losing lives, pregnancy, on death thoughts, and another rework of dark forest thoughts

## Why This Is Good For ClanGen

many categories I've edited only have one possibility which gets old especially for things you see often like announcing pregnancies or leaders losing lives plus dark forest thoughts being reworked for traits cause a nervous o compassionate cat wouldnt be suddenly evil as soon as they get to the dark forest, plus, the way it currently is, it feels a bit like whiplash for, say, a bloodthirsty cat to regret and be sobbing, then regret nothing, then back to crying and sad and regretting things


## Proof of Testing

![20240922_015232](https://github.com/user-attachments/assets/8d744b31-08e2-45e2-b2c7-152de18fad2f)
![20240922_014242](https://github.com/user-attachments/assets/6e410dcd-704b-46b8-927a-cab65b5e7777)
![20240922_015224](https://github.com/user-attachments/assets/f5f7f1bb-bef0-43f9-b70b-e40c4baddd47)
![20240922_015153](https://github.com/user-attachments/assets/36093973-bf32-4110-aed3-82a8d91aebf3)
![20240913_095415-1](https://github.com/user-attachments/assets/4d064fe7-3c2c-40f9-bdfe-d626ea4bfe95)
![20240922_015139](https://github.com/user-attachments/assets/f32faf95-6305-408f-ba81-744713e577e7)
![20240922_014303](https://github.com/user-attachments/assets/7ab1cb3e-eb75-4853-aace-84b3595b33fe)


## Changelog/Credits

reworked dark forest thoughts
added pregnancy content
added death content
-Fuzzy

Yzzuwyzzuf committed:
Update leader_life.json

Add 2 death lines

Yzzuwyzzuf committed:
Update leader_life.json

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update leader_death.json

Yzzuwyzzuf committed:
Update leader_life.json

Yzzuwyzzuf committed:
Merge branch 'ClanGenOfficial:development' into Ondeath-lines

Yzzuwyzzuf committed:
Update pregnancy.json

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update exiled.json

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Merge branch 'ClanGenOfficial:development' into life-and-death-update

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update elder.json

Yzzuwyzzuf committed:
Update exiled.json

Yzzuwyzzuf committed:
Update exiled.json

Yzzuwyzzuf committed:
Update pregnancy.json

Yzzuwyzzuf committed:
Update pregnancy.json

Yzzuwyzzuf committed:
Update pregnancy.json

Yzzuwyzzuf committed:
Update pregnancy.json

Yzzuwyzzuf committed:
Update pregnancy.json

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -1,6 +1,9 @@
   "announcement": [
-    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits."
+    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits.",
+	  "m_c had been secretive for a while now and nobody could even begin to guess what about... until today, that is when m_c announced {PRONOUN/m_c/poss} pregnancy!",
```
```suggestion
	  "m_c had been secretive for a while now and nobody could even begin to guess what about... until today when m_c announced {PRONOUN/m_c/poss} pregnancy!",
```

j-gynn replied:
```suggestion
	  "m_c had been secretive for a while now and nobody could even begin to guess what about... until today, that is, when m_c announced {PRONOUN/m_c/poss} pregnancy!",
```
both are possible, depends what you wanna say with it

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -1,6 +1,9 @@
-    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits."
+    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits.",
+	  "m_c had been secretive for a while now and nobody could even begin to guess what about... until today, that is when m_c announced {PRONOUN/m_c/poss} pregnancy!",
+          "The entire Clan could see the signs way before m_c did. Everyone urged {PRONOUN/m_c/object} to visit the medicine den to no avail. The Clan nearly gave up until finally, m_c announced {PRONOUN/m_c/poss} pregnancy.",
```
```suggestion
          "The entire Clan could see the signs way before m_c did. Everyone urged {PRONOUN/m_c/object} to visit the medicine den to no avail. They nearly gave up until, finally, m_c announced {PRONOUN/m_c/poss} pregnancy.",
```

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -1,6 +1,9 @@
+    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits.",
+	  "m_c had been secretive for a while now and nobody could even begin to guess what about... until today, that is when m_c announced {PRONOUN/m_c/poss} pregnancy!",
+          "The entire Clan could see the signs way before m_c did. Everyone urged {PRONOUN/m_c/object} to visit the medicine den to no avail. The Clan nearly gave up until finally, m_c announced {PRONOUN/m_c/poss} pregnancy.",
+	  "m_c's been worried sick over a recent illness {PRONOUN/m_c/subject} couldn't quite explain until a recent visit to the medicine den made everything all clear. m_c's pregnant!"
```
```suggestion
	  "m_c's been worried sick over a recent illness {PRONOUN/m_c/subject} couldn't quite explain, until a recent visit to the medicine den made everything clear. m_c's pregnant!"
```

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -16,7 +19,15 @@
+    "m_c hopes {PRONOUN/m_c/subject}'ll have a large litter. m_c always wanted a big family after all.",
+    "m_c is unsure how many kits {PRONOUN/m_c/subject}'ll have, nor {VERB/m_c/do/does} {PRONOUN/m_c/subject} care. As long as they're happy and healthy, that's all that matters.",
+    "m_c dosen't know or care how many kits {PRONOUN/m_c/subject}'ll have.",
+    "m_c, citing mothers' intuition, claims {PRONOUN/m_c/subject} know {PRONOUN/m_c/subject}'ll have a small litter.",
```
"know" needs to be verb tagged

keplerstelescope replied:
```suggestion
    "m_c, citing queen's intuition, claims {PRONOUN/m_c/subject} {VERB/m_c/know/knows} {PRONOUN/m_c/subject}'ll have a small litter.",
```
i don't think it should say "mother" since a male cat can also carry kits. i'm suggesting "queen" because i view it as a gender neutral word in Clan society, meaning "cat that is currently carrying/nursing kits", but if that's not an angle devs agree with, i'm happy for it to be edited further

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -16,7 +19,15 @@
+    "m_c is unsure how many kits {PRONOUN/m_c/subject}'ll have, nor {VERB/m_c/do/does} {PRONOUN/m_c/subject} care. As long as they're happy and healthy, that's all that matters.",
+    "m_c dosen't know or care how many kits {PRONOUN/m_c/subject}'ll have.",
+    "m_c, citing mothers' intuition, claims {PRONOUN/m_c/subject} know {PRONOUN/m_c/subject}'ll have a small litter.",
+    "m_c, citing mothers' intuition, claims {PRONOUN/m_c/subject} know {PRONOUN/m_c/subject}'ll have a big litter."
```
"know" needs to be verb tagged

keplerstelescope replied:
```suggestion
    "m_c, citing queen's intuition, claims {PRONOUN/m_c/subject} {VERB/m_c/know/knows}  {PRONOUN/m_c/subject}'ll have a big litter."
```
ditto to last suggestion

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -29,7 +40,10 @@
     " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} scolded into taking up a nest in the nursery, despite {PRONOUN/m_c/poss} best protests.",
-    " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} keen to move into the nursery quickly, as though by doing so {PRONOUN/m_c/poss} litter will get the message and appear sooner."
+    " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} keen to move into the nursery quickly, as though by doing so {PRONOUN/m_c/poss} litter will get the message and appear sooner.",
+	  " Despite {PRONOUN/m_c/object} protests, {PRONOUN/m_c/subject} has no choice but to settle in the nursery as nobody wants to let {PRONOUN/m_c/object} work til the kits arrive.",
```
"has" needs a verb tag

keplerstelescope replied:
```suggestion
    " Despite {PRONOUN/m_c/object} protests, {PRONOUN/m_c/subject} {VERB/m_c/have/has} no choice but to settle in the nursery, as nobody wants to let {PRONOUN/m_c/object} work til the kits arrive.",
```

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -29,7 +40,10 @@
     " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} scolded into taking up a nest in the nursery, despite {PRONOUN/m_c/poss} best protests.",
-    " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} keen to move into the nursery quickly, as though by doing so {PRONOUN/m_c/poss} litter will get the message and appear sooner."
+    " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} keen to move into the nursery quickly, as though by doing so {PRONOUN/m_c/poss} litter will get the message and appear sooner.",
+	  " Despite {PRONOUN/m_c/object} protests, {PRONOUN/m_c/subject} has no choice but to settle in the nursery as nobody wants to let {PRONOUN/m_c/object} work til the kits arrive.",
```
```suggestion
	  " Despite {PRONOUN/m_c/object} protests, {PRONOUN/m_c/subject} has no choice but to settle in the nursery, nobody wants to let {PRONOUN/m_c/object} work til the kits arrive.",
```

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -29,7 +40,10 @@
+    " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} keen to move into the nursery quickly, as though by doing so {PRONOUN/m_c/poss} litter will get the message and appear sooner.",
+	  " Despite {PRONOUN/m_c/object} protests, {PRONOUN/m_c/subject} has no choice but to settle in the nursery as nobody wants to let {PRONOUN/m_c/object} work til the kits arrive.",
+	  " {PRONOUN/m_c/subject/CAP} couldn't settle into the nursery fast enough.",
+	  " So many things could happen to m_c and {PRONOUN/m_c/poss} kits if {PRONOUN/m_c/subject} kept working, no way did {PRONOUN/m_c/subject} want anything to happen to them so into the nursery {PRONOUN/m_c/subject} go."
```
```suggestion
	  " So many things could happen to m_c and {PRONOUN/m_c/poss} kits if {PRONOUN/m_c/subject} kept working, no way did {PRONOUN/m_c/subject} want anything to happen to them, so into the nursery {PRONOUN/m_c/subject} go."
```

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -38,7 +52,11 @@
     " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} still full of energy and moving to the nursery would feel stifling - maybe next moon.",
-    " Surely it's not time to move into the nursery, though."
+    " Surely it's not time to move into the nursery, though.",
+	  " {PRONOUN/m_c/subject/CAP} couldn't imagine being trapped in the nursery yet though and insists {PRONOUN/m_c/subject} can keep working.",
```
```suggestion
	  " {PRONOUN/m_c/subject/CAP} couldn't imagine being trapped in the nursery yet and insists {PRONOUN/m_c/subject} can keep working.",
```
"insists" needs a verb tag

keplerstelescope replied:
```suggestion
    " {PRONOUN/m_c/subject/CAP} couldn't imagine being trapped in the nursery yet, and {VERB/m_c/insist/insists} {PRONOUN/m_c/subject} can keep working.",
```

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -38,7 +52,11 @@
-    " Surely it's not time to move into the nursery, though."
+    " Surely it's not time to move into the nursery, though.",
+	  " {PRONOUN/m_c/subject/CAP} couldn't imagine being trapped in the nursery yet though and insists {PRONOUN/m_c/subject} can keep working.",
+	  " {PRONOUN/m_c/subject/CAP} dare any cat to force {PRONOUN/m_c/object} into that stuffy old nursery and, to {PRONOUN/m_c/object} delight, no cat wanted to. Maybe next moon but not now.",
```
```suggestion
	  " {PRONOUN/m_c/subject/CAP} dare any cat to force {PRONOUN/m_c/object} into that stuffy old nursery and, to {PRONOUN/m_c/object} delight, no cat wanted to. Maybe next moon, but not now.",
```

keplerstelescope replied:
```suggestion
	  " {PRONOUN/m_c/subject/CAP} {VERB/m_c/dare/dares} any cat to force {PRONOUN/m_c/object} into that stuffy old nursery and, to {PRONOUN/m_c/object} delight, no cat tries to. Maybe next moon, but not now.",
```

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -38,7 +52,11 @@
+	  " {PRONOUN/m_c/subject/CAP} couldn't imagine being trapped in the nursery yet though and insists {PRONOUN/m_c/subject} can keep working.",
+	  " {PRONOUN/m_c/subject/CAP} dare any cat to force {PRONOUN/m_c/object} into that stuffy old nursery and, to {PRONOUN/m_c/object} delight, no cat wanted to. Maybe next moon but not now.",
+	  " {PRONOUN/m_c/object/CAP} wouldn't dare settle down and let others work while {PRONOUN/m_c/subject} rested! Things continue as usual for now.",
+	  " Theres so much that needs to be done though! {PRONOUN/m_c/subject/CAP} just can't settle down yet!"
```
```suggestion
	  " There's so much that needs to be done! {PRONOUN/m_c/subject/CAP} just can't settle down yet!"
```

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -101,7 +124,8 @@
       "Though all looks fine, the Clan will wake tomorrow to discover m_c's body cold in the nursery, {PRONOUN/m_c/poss} {insert} mewing in vain for their parent.",
-      "Though birth is always considered a difficult and risky event, no one thought they'd lose m_c to it. Not after the {insert} was all born and seemed fine... they thought the blood loss was under control."
+      "Though birth is always considered a difficult and risky event, no one thought they'd lose m_c to it. Not after the {insert} was all born and seemed fine... they thought the blood loss was under control.",
+	    "Everyone knew m_c was the most excited of everyone to meet {PRONOUN/m_c/poss} kits... It was a cruel twist of fate however, as m_c's body lay cooling leaving the {insert} without {PRONOUN/m_c/object}."
```
```suggestion
	    "Everyone knew m_c was the most excited of everyone to meet {PRONOUN/m_c/poss} kits... It's a cruel twist of fate, however, as m_c's body lays cooling, leaving the {insert} without {PRONOUN/m_c/object}."
```

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -112,7 +136,9 @@
-      "m_c chokes down the herbs given to {PRONOUN/m_c/object}, retching at the taste but knowing {PRONOUN/m_c/subject} {VERB/m_c/need/needs} them to stop the blood loss. "
+      "m_c chokes down the herbs given to {PRONOUN/m_c/object}, retching at the taste but knowing {PRONOUN/m_c/subject} {VERB/m_c/need/needs} them to stop the blood loss.",
+	    "m_c wonders if {PRONOUN/m_c/poss} body will ever recover from the painful birth and blood loss.",
+	    "It took the medicine cat everything they had but m_c just about pulled through."
```
Unfortunately we can't mention the med cat here as we can't guarantee that they exist (nor can we pronoun tag for them)

keplerstelescope replied:
```suggestion
      "There were complications - painful ones - but m_c just about pulled through."
```

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -90,7 +109,11 @@
       "r_c goes to visit m_c in the nursery with {PRONOUN/m_c/poss} new {insert}, on a completely innocent mission to deliver food to the new parent.",
-      "The newly arrived {insert} that m_c has just given birth to looks suspiciously like r_c. "
+      "The newly arrived {insert} that m_c has just given birth to looks suspiciously like r_c. ",
+	    "No one wanted to ruin such a happy occasion but... it didn't take a genius to notice how m_c's {insert} looked suspiciously like r_c when they shouldn't...",
```
```suggestion
	    "No one wanted to ruin such a happy occasion but... it didn't take a genius to notice how m_c's {insert} looked suspiciously like r_c...",
```

keplerstelescope replied:
```suggestion
	    "No one wants to ruin such a happy occasion, but... it doesn't take a genius to notice how m_c's {insert} looks suspiciously like r_c...",
```
i think these are supposed to be in present tense?

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -90,7 +109,11 @@
+      "The newly arrived {insert} that m_c has just given birth to looks suspiciously like r_c. ",
+	    "No one wanted to ruin such a happy occasion but... it didn't take a genius to notice how m_c's {insert} looked suspiciously like r_c when they shouldn't...",
+	    "m_c can see it, m_c's mate can see it, r_c can see it, everyone in c_n can see the new {insert} look too much like r_c to be coincidence...",
+        "Ever since the birth, rumor's gone around that m_c's {insert} belong to r_c's and not {PRONOUN/m_c/poss} mate and while no one has confirmed anything, no one really has to when you just look at them...",
```
```suggestion
        "Ever since the birth, rumor's gone around that m_c's {insert} belongs to r_c.  While no one has confirmed anything, no one really has to when the evidence is mewling right there...",
```
Some edits for better flow.

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -90,7 +109,11 @@
-      "The newly arrived {insert} that m_c has just given birth to looks suspiciously like r_c. "
+      "The newly arrived {insert} that m_c has just given birth to looks suspiciously like r_c. ",
+	    "No one wanted to ruin such a happy occasion but... it didn't take a genius to notice how m_c's {insert} looked suspiciously like r_c when they shouldn't...",
+	    "m_c can see it, m_c's mate can see it, r_c can see it, everyone in c_n can see the new {insert} look too much like r_c to be coincidence...",
```
```suggestion
	    "m_c can see it, m_c's mate can see it, r_c can see it, everyone in c_n can see that the new {insert} look too much like r_c to be coincidence...",
```

keplerstelescope replied:
```suggestion
	    "m_c can see it, m_c's mate can see it, r_c can see it, everyone in c_n can see that the new {insert} looks too much like r_c for it to be coincidence...",
```

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -90,7 +109,11 @@
+	    "No one wanted to ruin such a happy occasion but... it didn't take a genius to notice how m_c's {insert} looked suspiciously like r_c when they shouldn't...",
+	    "m_c can see it, m_c's mate can see it, r_c can see it, everyone in c_n can see the new {insert} look too much like r_c to be coincidence...",
+        "Ever since the birth, rumor's gone around that m_c's {insert} belong to r_c's and not {PRONOUN/m_c/poss} mate and while no one has confirmed anything, no one really has to when you just look at them...",
+        "m_c's mate is such a wonderful cat deserving of being a parent... so why does m_c's {insert} look more like r_c instead?"
```
```suggestion
        "m_c's mate is such a wonderful cat, deserving of being a parent... so why does m_c's {insert} look more like r_c instead?"
```

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -123,7 +149,9 @@
 		"Some say m_c didn't make it.",
-		"Some say m_c would have needed a medicine cat to survive."
+		"Some say m_c would have needed a medicine cat to survive.",
+		"Hopefully those kits have someone to care for them since m_c apparently didn't make it...",
```
```suggestion
		"Hopefully those kits have someone to care for them, as m_c apparently didn't make it...",
```

scribblecrumb commented on resources/dicts/thoughts/dead/darkforest/general.json:
```diff
@@ -60,10 +46,100 @@
+            "Bought {PRONOUN/m_c/self} some time to hide after talking {PRONOUN/m_c/poss} way out of training",
+            "Is wondering if everything was really worth it...",
+            "Is fleeing from another member of the Dark Forest",
+            "Wants nothing more then to fade away...",
```
```suggestion
            "Wants nothing more than to fade away...",
```

scribblecrumb commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -223,12 +247,61 @@
+        "id": "dead_parent_to_alive_child",
+        "thoughts":[
+            "Snuck into r_c's dream to comfort {PRONOUN/r_c/object} after a hard day",
+            "Can't stop fretting over r_c even though there's nothing {PRONOUN/m_c/subject} can do",
```
```suggestion
            "Can't stop fretting over r_c, even though there's nothing {PRONOUN/m_c/subject} can do",
```

scribblecrumb commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -223,12 +247,61 @@
+            "Is talking the ears off any cat willing to listen to {PRONOUN/m_c/object} go on and on and on about r_c",
+            "Is feeling guilty watching r_c miss {PRONOUN/m_c/object}",
+            "Deeply regrets leaving r_c without {PRONOUN/m_c/object}",
+            "Visited r_c's dream to tell {PRONOUN/r_c/object} just how immensely proud {PRONOUN/m_c/subject} {VERB/m_c/are/is} of {PRONOUN/r_c/subject}"
```
Could we get this constrained to STAR and DREAM skills?

scribblecrumb commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -223,12 +247,61 @@
+        "id": "dead_outsider_parent_to_alive_child",
+        "thoughts":[
+            "Regrets not joining a Clan if it meant being able to raise r_c",
+            "Still doesn't trust Clan cats but is happy r_c is safe",
```
```suggestion
            "Still doesn't trust Clan cats, but is happy r_c is safe",
```

scribblecrumb commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -223,12 +247,61 @@
-            "Is watching r_c play"
+            "Is watching r_c play",
+            "Snuck into r_c's dreams to play with {PRONOUN/r_c/object} one last time",
+            "Visits r_c in {PRONOUN/r_c/poss} to tell them to live for the both of them",
```
```suggestion
            "Visits r_c in {PRONOUN/r_c/poss} dreams to tell them to live for the both of them",
```
again, could these dream focused ones be moved to STAR and DREAM skill constraints?

scribblecrumb commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -340,7 +413,10 @@
-            "Is proud of who r_c has become"
+            "Is proud of who r_c has become",
+            "Visited r_c's dreams recently and is still buzzing about it",
+            "Wants to visit r_c in {PRONOUN/r_c/poss} dreams cause {PRONOUN/m_c/subject} miss being with {PRONOUN/r_c/object} so much",
```
```suggestion
            "Wants to visit r_c in {PRONOUN/r_c/poss} dreams, as {PRONOUN/m_c/subject} miss being with {PRONOUN/r_c/object} so much",
```

scribblecrumb commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -340,7 +413,10 @@
             "Celebrates r_c's successes with {PRONOUN/r_c/object}",
-            "Is proud of who r_c has become"
+            "Is proud of who r_c has become",
+            "Visited r_c's dreams recently and is still buzzing about it",
```
move to STAR and DREAM skill constraints

scribblecrumb commented on resources/dicts/thoughts/ondeath/darkforest/leader_life.json:
```diff
@@ -2,7 +2,10 @@
+			"Was startled to find {PRONOUN/m_c/self} in the Dark Forest for a moment... did {PRONOUN/m_c/subject} just lose a life?",
+			"Has been somber ever since {PRONOUN/m_c/subject} lost a life...",
+			"Has been thinking of the cat who gave {PRONOUN/m_c/object} the life {PRONOUN/m_c/subject} just lost...",
+			"Is feeling a sense of dread. If {PRONOUN/m_c/subject} ended up in the Dark Forest... what kind of legacy are {PRONOUN/m_c/subject} leading?"
```
```suggestion
			"Is feeling a sense of dread. If {PRONOUN/m_c/subject} ended up in the Dark Forest... what kind of legacy are {PRONOUN/m_c/subject} creating?"
```

keplerstelescope commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -16,7 +19,15 @@
+    "m_c thinks {PRONOUN/m_c/subject}'ll have a large litter.",
+    "m_c hopes {PRONOUN/m_c/subject}'ll have a large litter. m_c always wanted a big family after all.",
+    "m_c is unsure how many kits {PRONOUN/m_c/subject}'ll have, nor {VERB/m_c/do/does} {PRONOUN/m_c/subject} care. As long as they're happy and healthy, that's all that matters.",
+    "m_c dosen't know or care how many kits {PRONOUN/m_c/subject}'ll have.",
```
```suggestion
    "m_c doesn't know or care how many kits {PRONOUN/m_c/subject}'ll have.",
```

keplerstelescope commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -38,7 +52,11 @@
+    " Surely it's not time to move into the nursery, though.",
+	  " {PRONOUN/m_c/subject/CAP} couldn't imagine being trapped in the nursery yet though and insists {PRONOUN/m_c/subject} can keep working.",
+	  " {PRONOUN/m_c/subject/CAP} dare any cat to force {PRONOUN/m_c/object} into that stuffy old nursery and, to {PRONOUN/m_c/object} delight, no cat wanted to. Maybe next moon but not now.",
+	  " {PRONOUN/m_c/object/CAP} wouldn't dare settle down and let others work while {PRONOUN/m_c/subject} rested! Things continue as usual for now.",
```
```suggestion
	  " {PRONOUN/m_c/object/CAP} wouldn't dare settle down and let others work while {PRONOUN/m_c/subject} rested! Things'll continue as usual for now.",
```
minor nitpick

keplerstelescope commented on resources/dicts/thoughts/dead/darkforest/exiled.json:
```diff
@@ -3,17 +3,18 @@
+            "Is watching {PRONOUN/m_c/poss} old Clan with scorn, hoping for an opportunity to get back at them",
             "Is cursing {PRONOUN/m_c/poss} old Clan"
         ],
     "main_trait_constraint": [
```
could potentially add strict?

keplerstelescope commented on resources/dicts/thoughts/dead/darkforest/general.json:
```diff
@@ -60,10 +46,100 @@
+        "thoughts": [
+            "Wishes {PRONOUN/m_c/subject} could go to StarClan",
+            "Feels regret for {PRONOUN/m_c/poss} actions",
+            "Is curled up in a ball crying",
```
```suggestion
            "Is curled up in a ball, crying",
```

keplerstelescope commented on resources/dicts/thoughts/dead/darkforest/general.json:
```diff
@@ -60,10 +46,100 @@
+            "Wants to unleash mayhem on the cats who hurt {PRONOUN/m_c/object} in life",
+            "Is scheming",
+            "Found a living cat that {PRONOUN/m_c/subject} could manipulate",
+            "Found a living cat that {PRONOUN/m_c/subject} {VERB/m_c/hope/hopes} will help {PRONOUN/m_c/poss} plans come to fruition",
```
```suggestion
            "Found a living cat that, hopefully, will help {PRONOUN/m_c/poss} plans come to fruition",
```
sentence felt stilted

keplerstelescope commented on resources/dicts/thoughts/dead/starclan/exiled.json:
```diff
@@ -1,3 +1,9 @@
+    {
+        "id": "exiled_starclan_thoughts",
+        "thoughts": [
+            "Truly believed {PRONOUN/m_c/subject} {VERB/m_c/were/was} going to the Dark Forest ever since {PRONOUN/m_c/subject} {VERB/m_c/were/was} exiled",
```
```suggestion
            "Truly believed {PRONOUN/m_c/subject} {VERB/m_c/were/was} going to the Dark Forest after {PRONOUN/m_c/poss} exile",
```
nitpick?

Yzzuwyzzuf replied:
I think I still prefer my wording. I feel like this wording best gets the idea that the cat has been thinking of it the entire time

keplerstelescope commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -69,7 +70,8 @@
             "Is speaking with r_c about how c_n came to be",
-            "Is telling other StarClan cats stories of c_n's beginning"
+            "Is telling other StarClan cats stories of c_n's beginning",
+            "Is telling other StarClan cats the ambitions, hopes, and dreams {PRONOUN/m_c/subject} had for c_n"
```
```suggestion
            "Is telling other StarClan cats about the ambitions, hopes, and dreams {PRONOUN/m_c/subject} had for c_n"
```

keplerstelescope commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -123,6 +125,27 @@
+        ],
+         "random_trait_constraint": [
+             "arrogant",
+             "bloodthirsty",
```
```suggestion
             "bloodthirsty",
             "cold",
```

keplerstelescope commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -165,7 +188,8 @@
             "Sends a vision of an undiscovered herb patch to r_c",
-            "Lets {PRONOUN/m_c/poss} scent linger near r_c, if only for a moment"
+            "Lets {PRONOUN/m_c/poss} scent linger near r_c, if only for a moment",
+            "Tells r_c where to find an herb c_n is low on"
```
```suggestion
            "Leads r_c to a patch of an herb c_n is low on"
```
i think it should be more subtle than the original, but i'm not totally confident in the phrasing of this edit. could be better?

keplerstelescope commented on resources/dicts/thoughts/ondeath/darkforest/general.json:
```diff
@@ -2,7 +2,19 @@
+        "id": "star_death_dark_forest",
+        "thoughts": [
+            "Is shocked to find {PRONOUN/m_c/self} here and not StarClan",
+            "Just can't believe {PRONOUN/m_c/subject} ended up in the Dark Forest. Was StarClan not watching out for c_n?"
```
```suggestion
            "Just can't believe {PRONOUN/m_c/subject} ended up in the Dark Forest. Was StarClan not watching over c_n?"
```

keplerstelescope reviewed (commented):
edits! there's some formatting issues i'd like to solve but it'd be near-impossible to do in a review like this -.-

Yzzuwyzzuf committed:
Update resources/dicts/conditions/pregnancy.json

Co-authored-by: J Gynn <48025294+j-gynn@users.noreply.github.com>

Yzzuwyzzuf committed:
Update pregnancy.json

edit made with all suggested changes in mind

keplerstelescope commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -29,7 +40,10 @@
     " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} scolded into taking up a nest in the nursery, despite {PRONOUN/m_c/poss} best protests.",
-    " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} keen to move into the nursery quickly, as though by doing so {PRONOUN/m_c/poss} litter will get the message and appear sooner."
+    " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} keen to move into the nursery quickly, as though by doing so {PRONOUN/m_c/poss} litter will get the message and appear sooner.",
+	  " Despite {PRONOUN/m_c/object} protests, {PRONOUN/m_c/subject} {PRONOUN/m_c/has/have} no choice but to settle in the nursery, nobody wants to let {PRONOUN/m_c/object} work til the kits arrive.",
```
```suggestion
    " Despite {PRONOUN/m_c/poss} protests, {PRONOUN/m_c/subject} {VERB/m_c/have/has} no choice but to settle in the nursery; nobody wants to let {PRONOUN/m_c/object} work til after the kits arrive.",
```

keplerstelescope commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -16,7 +19,15 @@
+    "m_c is unsure how many kits {PRONOUN/m_c/subject}'ll have, nor {VERB/m_c/do/does} {PRONOUN/m_c/subject} care. As long as they're happy and healthy, that's all that matters.",
+    "m_c dosen't know or care how many kits {PRONOUN/m_c/subject}'ll have.",
+    "m_c, citing queens' intuition, claims {PRONOUN/m_c/subject} {PRONOUN/m_c/know/knows} {PRONOUN/m_c/subject}'ll have a small litter.",
+    "m_c, citing queens' intuition, claims {PRONOUN/m_c/subject} {PRONOUN/m_c/know/knows} {PRONOUN/m_c/subject}'ll have a big litter."
```
```suggestion
    "m_c, citing queens' intuition, claims {PRONOUN/m_c/subject} {VERB/m_c/know/knows} {PRONOUN/m_c/subject}'ll have a big litter."
```

keplerstelescope commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -16,7 +19,15 @@
+    "m_c hopes {PRONOUN/m_c/subject}'ll have a large litter. m_c always wanted a big family after all.",
+    "m_c is unsure how many kits {PRONOUN/m_c/subject}'ll have, nor {VERB/m_c/do/does} {PRONOUN/m_c/subject} care. As long as they're happy and healthy, that's all that matters.",
+    "m_c dosen't know or care how many kits {PRONOUN/m_c/subject}'ll have.",
+    "m_c, citing queens' intuition, claims {PRONOUN/m_c/subject} {PRONOUN/m_c/know/knows} {PRONOUN/m_c/subject}'ll have a small litter.",
```
```suggestion
    "m_c, citing queens' intuition, claims {PRONOUN/m_c/subject} {VERB/m_c/know/knows} {PRONOUN/m_c/subject}'ll have a small litter.",
```
i have no idea if tagging verbs as pronouns was me specifically messing up, but whoops!

Yzzuwyzzuf replied:
Was going crazy tryna find where I messed up! Ty!

keplerstelescope reviewed (commented):
found a big whoops!

Yzzuwyzzuf committed:
Update pregnancy.json

Yzzuwyzzuf committed:
Update pregnancy.json

Yzzuwyzzuf committed:
Update exiled.json

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/dead/starclan/general.json

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update leader_life.json

Yzzuwyzzuf reviewed (commented):
I believe I've fixed all issues but please let me know if I missed anything! ^-^

Yzzuwyzzuf committed:
Update pregnancy.json

Yzzuwyzzuf committed:
Update resources/dicts/conditions/pregnancy.json

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

scribblecrumb commented:
Go ahead and resolve all conversations that you've addressed! Makes it easier for me to navigate things.

selkirks commented on resources/dicts/thoughts/ondeath/darkforest/leader_life.json:
```diff
@@ -2,7 +2,10 @@
+			"Was startled to find {PRONOUN/m_c/self} in the Dark Forest for a moment... did {PRONOUN/m_c/subject} just lose a life?",
+			"Has been somber ever since {PRONOUN/m_c/subject} lost a life...",
+			"Has been thinking of the cat who gave {PRONOUN/m_c/object} the life {PRONOUN/m_c/subject} just lost...",
+			"Is feeling a sense of dread. If {PRONOUN/m_c/subject} ended up in the Dark Forest... what kind of legacy are {PRONOUN/m_c/subject} creating?"
```
verb tag 'are'
reads as both 'what kind of legacy are they creating?' and 'what kind of legacy are she creating?'

Yzzuwyzzuf replied:
I think I got it but knowing my luck, I put it in backwards lol

selkirks commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -335,12 +452,30 @@
             "Celebrates r_c's successes with {PRONOUN/r_c/object}",
-            "Is proud of who r_c has become"
+            "Is proud of who r_c has become",
+            "Wants to visit r_c in {PRONOUN/r_c/poss} dreams, as {PRONOUN/m_c/subject} miss being with {PRONOUN/r_c/object} so much",
```
verb tag 'miss'
reads as both 'as they miss being with them so much' and 'as she miss being with them so much'

Yzzuwyzzuf replied:
Fixed by splitting the thought into 2 lol now the tagging isn't needed

selkirks commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -224,16 +249,108 @@
+        "id": "dead_child_to_alive_child_sibling2",
+        "thoughts":[
+            "Snuck into r_c's dreams to play with {PRONOUN/r_c/object} one last time",
+            "Visits r_c in {PRONOUN/r_c/poss} to tell them to live for the both of them"
```
need something after {PRONOUN/r_c/poss}
reads as 'visits r_c in their to tell them to live...'

selkirks commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -224,16 +249,108 @@
             "Wishes {PRONOUN/m_c/subject} could play with r_c",
-            "Is watching r_c play"
+            "Is watching r_c play",
+            "Is watching over r_c reminicing over the short time they had together"
```
probably could use a comma after the first r_c, and reminiscing is mispelled

selkirks commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -224,16 +249,108 @@
+    {
+        "id": "dead_outsider_parent_to_alive_child",
+        "thoughts":[
+            "Regrets not joining a Clan if it meant being able to raise r_c",
```
no edit this just makes me sad :( i love it

selkirks commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -224,16 +249,108 @@
+            "Regrets not joining a Clan if it meant being able to raise r_c",
+            "Still doesn't trust Clan cats, but is happy r_c is safe",
+            "Is just glad r_c was saved from meeting the same fate {PRONOUN/m_c/subject} did",
+            "Is so greatful to c_n for saving r_c"
```
'greatful' should be 'grateful'

selkirks commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -224,16 +249,108 @@
+        "id": "dead_parent_to_alive_child2",
+        "thoughts":[
+            "Snuck into r_c's dream to comfort {PRONOUN/r_c/object} after a hard day",
+            "Visited r_c's dream to tell {PRONOUN/r_c/object} just how immensely proud {PRONOUN/m_c/subject} {VERB/m_c/are/is} of {PRONOUN/r_c/subject}"
```
the last {PRONOUN/r_c/subject} should be {PRONOUN/r_c/object} instead

selkirks commented on resources/dicts/thoughts/dead/starclan/general.json:
```diff
@@ -165,7 +189,8 @@
             "Sends a vision of an undiscovered herb patch to r_c",
-            "Lets {PRONOUN/m_c/poss} scent linger near r_c, if only for a moment"
+            "Lets {PRONOUN/m_c/poss} scent linger near r_c, if only for a moment",
+            "Lead r_c to a hidden patch of herbs the Clan is running low on"
```
'lead' should be 'leads'

selkirks commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -29,7 +40,10 @@
     " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} scolded into taking up a nest in the nursery, despite {PRONOUN/m_c/poss} best protests.",
-    " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} keen to move into the nursery quickly, as though by doing so {PRONOUN/m_c/poss} litter will get the message and appear sooner."
+    " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} keen to move into the nursery quickly, as though by doing so {PRONOUN/m_c/poss} litter will get the message and appear sooner.",
+	  " Despite {PRONOUN/m_c/object} protests, {PRONOUN/m_c/subject} {VERB/m_c/has/have} no choice but to settle in the nursery, nobody wants to let {PRONOUN/m_c/object} work til the kits arrive.",
```
the first {PRONOUN/m_c/object} shld be {PRONOUN/m_c/poss}

selkirks replied:
has and have shld be reversed
it shld look like
{VERB/m_c/have/has}

selkirks commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -1,6 +1,9 @@
+    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits.",
+	  "m_c had been secretive for a while now and nobody could even begin to guess what about... until today, that is, when m_c announced {PRONOUN/m_c/poss} pregnancy!",
+          "The entire Clan could see the signs way before m_c did. Everyone urged {PRONOUN/m_c/object} to visit the medicine den to no avail. They just about gave up until, finally, m_c announced {PRONOUN/m_c/poss} pregnancy.",
+	  "m_c's been worried sick over a recent illness {PRONOUN/m_c/subject} couldn't quite explain, until a recent visit to the medicine den made everything clear. m_c's pregnant!"
```
should be
'medicine cat den' not 'medicine den'

selkirks commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -1,6 +1,9 @@
-    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits."
+    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits.",
+	  "m_c had been secretive for a while now and nobody could even begin to guess what about... until today, that is, when m_c announced {PRONOUN/m_c/poss} pregnancy!",
+          "The entire Clan could see the signs way before m_c did. Everyone urged {PRONOUN/m_c/object} to visit the medicine den to no avail. They just about gave up until, finally, m_c announced {PRONOUN/m_c/poss} pregnancy.",
```
should be 'medicine cat den' not 'medicine den'

Yzzuwyzzuf replied:
Last time I made a pr referring to the location, it was requested I change it to medicine den

Yzzuwyzzuf committed:
Update leader_life.json

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update pregnancy.json

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -1,6 +1,9 @@
-    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits."
+    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits.",
+	  "m_c had been secretive for a while now and nobody could even begin to guess what about... until today, that is, when m_c announced {PRONOUN/m_c/poss} pregnancy!",
+          "The entire Clan could see the signs way before m_c did. Everyone urged {PRONOUN/m_c/object} to visit the medicine den to no avail. They just about gave up until, finally, m_c announced {PRONOUN/m_c/poss} pregnancy.",
```
```suggestion
          "The entire Clan could see the signs way before m_c did. Everyone urged {PRONOUN/m_c/object} to visit the medicine cat den to no avail. They just about gave up until, finally, m_c announced {PRONOUN/m_c/poss} pregnancy.",
```
I would suggest just doing a ctrl+shift+r to find and replace all the "medicine den"s with "medicine cat den"

Yzzuwyzzuf replied:
i could have sworn it was medicine den, not medicine cats den. i definitely remember another pr i submitted where the inverse was requested

scribblecrumb commented on resources/dicts/conditions/pregnancy.json:
```diff
@@ -38,7 +52,11 @@
+    " Surely it's not time to move into the nursery, though.",
+	  " {PRONOUN/m_c/subject/CAP} couldn't imagine being trapped in the nursery yet and {VERB/m_c/insist/insists} {PRONOUN/m_c/subject} can keep working.",
+	  " {PRONOUN/m_c/subject/CAP} {VERB/m_c/dare/dares} any cat to force {PRONOUN/m_c/object} into that stuffy old nursery and, to {PRONOUN/m_c/object} delight, no cat tries to. Maybe next moon, but not now.",
+	  " {PRONOUN/m_c/object/CAP} wouldn't dare settle down and let others work while {PRONOUN/m_c/subject} rested! Things continue as usual for now.",
```
```suggestion
	  " {PRONOUN/m_c/object/CAP} wouldn't dare settle down and let others work while {PRONOUN/m_c/subject} rested! Things will continue as usual for now.",
```

Yzzuwyzzuf committed:
Update resources/dicts/conditions/pregnancy.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

Yzzuwyzzuf committed:
Update pregnancy.json

moved a thing I placed in the wrong category. did not change the medicine cat den thing. reasoning in comment left on suggestion

Yzzuwyzzuf committed:
Update pregnancy.json

fixed med cat den thing!
